computation/nlp/install-fuseki.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+    echo -e "\nUsage:\n$0 [ontology_filepath] \n"
+    exit 1
+fi
+
+echo "Creating server environment (for testing only)..."
+echo "Downloading fuseki binaries..."
+wget http://mirrors.ircam.fr/pub/apache/jena/binaries/apache-jena-fuseki-2.4.1.tar.gz
+#cp ~/Downloads/apache-jena-fuseki-2.4.1.zip ./
+echo "Extracting server..."
+tar -xvf apache-jena-fuseki-2.4.1.tar.gz
+#unzip apache-jena-fuseki-2.4.1.zip
+cd apache-jena-fuseki-2.4.1/
+echo "Starting empty server"
```

Comment:
When doing a script like this, it may be considered good usage to add a blank line before the echos, for readability purpose.